include/xsimd/types/xsimd_base.hpp

```diff
@@ -1777,6 +1783,36 @@ namespace xsimd
         return kernel::select(cond, a(), b());
     }
 
+    /**
+     * Unpack and interleave data from the LOW half of batches \c lhs and \c rhs.
+     * Store the results in the Return value.
+     * @param lhs a batch of integer or floating point or double precision values.
+     * @param rhs a batch of integer or floating point or double precision values.
+     * @return a batch of the low part of shuffled values.
+     */
+    template <class X>
+    inline batch_type_t<X> zip_lo(const simd_base<X>& lhs, const simd_base<X>& rhs)
+    {
+        using value_type = typename simd_batch_traits<X>::value_type;
+        using kernel = detail::batch_kernel<value_type, simd_batch_traits<X>::size>;
+        return kernel::zip_lo(lhs(), rhs());
```

Comment:
This requires to define `zip_lo` for all `batch_kernel` classes, otherwise the code may fail to compile if the user calls `zip_lo` (or maybe the PR is still in progress?)